src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -1230,7 +1232,7 @@ class CopyLambdaProxyClassInfoToArchive : StackObj {
     // The following check has been moved to SystemDictionaryShared::check_excluded_classes(), which
     // happens before the classes are copied.
     //
-    // if (SystemDictionaryShared::is_excluded_class(info._proxy_klasses->at(0))) {
+    // if (SystemDictionaryShared::should_be_excluded_class(info._proxy_klasses->at(0))) {
```

Comment:
The change seems unrelated to the PR. We don't have a function named `SystemDictionaryShared::should_be_excluded_class`

Reply:
This is a mistake -- should not be changed.